beluga/include/beluga/motion/differential_drive_model.hpp

```diff
@@ -123,7 +123,6 @@ class DifferentialDriveModel : public Mixin {
               ? Sophus::SO2d{std::atan2(translation.y(), translation.x())} * previous_orientation.inverse()
               : Sophus::SO2d{0.0};
```

Comment:
nit: from my experience with perf, `SO2d{0.0}` does a bunch of caculations to determine real and imaginary. `SO2d{}` is much faster because it just sets the right values. Not that it makes a lot of difference here.

Reply:
Addressed in https://github.com/Ekumen-OS/beluga/pull/267/commits/2ed0708d68dda5bffd9d0274cc7c74e8cc440e4b.